api/src/certification/session-management/infrastructure/repositories/assessment-result-jury-comment-repository.js

```diff
@@ -0,0 +1,18 @@
+import { knex } from '../../../../../db/knex-database-connection.js';
+import { AssessmentResultJuryComment } from '../../domain/models/AssessmentResultJuryComment.js';
+
+const getLatestAssessmentResult = async function ({ certificationCourseId }) {
```

Comment:
```suggestion
const getLatestAssessmentResultJuryComment = async function ({ certificationCourseId }) {
```

On retourne plutôt le dernier `jury comment` à la place d'un `assessment result`, même si cet entité n'a pas vraiment de vie sans `assessment-result`.